cloudbuild.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
```

Comment:
Could you add a comment like this here? Thanks.

This config is provided for development/testing purposes only.
The official Docker image of stackdriver-prometheus-sidecar is not built
using this config.

Reply:
Added.